recipes/dlib/osx_jpeg.patch

```diff
@@ -0,0 +1,12 @@
+--- dlib/CMakeLists.txt
++++ dlib/CMakeLists.txt
+@@ -362,7 +362,7 @@
+             set(CMAKE_REQUIRED_LIBRARIES ${JPEG_LIBRARY})
+             CHECK_FUNCTION_EXISTS(jpeg_read_header LIBJPEG_IS_GOOD)
+          endif()
+-         if (JPEG_FOUND AND LIBJPEG_IS_GOOD AND NOT APPLE)
++         if (JPEG_FOUND AND LIBJPEG_IS_GOOD)
```

Comment:
This can actually probably be put upstream because I don't think this is an issue for modern OSX machines - unless we try and check for the OSX build before disabling it automatically.